common/autoware_auto_geometry/include/geometry/bounding_box/lfit.hpp

```diff
@@ -18,6 +18,8 @@
 /// \brief This file implements 2D pca on a linked list of points to estimate an oriented
 ///        bounding box
 
+// cspell: ignore LFIT, lfit
+
```

Comment:
I think LFIT maybe means "L-Shape Fitting"
```suggestion
// LFIT means "L-Shape Fitting"
```

Reply:
Thanks, I wanted to have such an information!